### PR TITLE
encrypt / decrypt

### PR DESCRIPTION
**Pull request by futurepaul**
alright this is looking good imo. I tested setting a password, removing the password, exporting with / without encryption

### CODE REVIEW

**Comment by futurepaul** [comment]
- [ ] make it so you can get rid of the password
- [ ] make sure it works with export

**Comment by futurepaul** [comment]
getting an error when I try to encrypt with no existing password. pretty sure it's not my fault, was working before

Screenshot 2023-07-08 at 6 47 40 PM

File: src/routes/settings/Encrypt.tsx [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+    const [store, _actions] = useMegaStore();
+    const [error, setError] = createSignal<Error>();
+    const [loading, setLoading] = createSignal(false);
+
+    const [_encryptPasswordForm, { Form, Field }] =
+        createForm<EncryptPasswordForm>({
+            initialValues: {
+                existingPassword: "",
+                password: "",
+                confirmPassword: ""
```
Comment (TonyGiorgio) [inline src/routes/settings/Encrypt.tsx:37]: Use `undefined`

File: src/logic/mutinyWalletSetup.ts [hunk 2]
```diff
@@ -138,7 +140,7 @@ export async function setupMutinyWallet(
     console.log("Using subscriptions address", subscriptions);
     const mutinyWallet = await new MutinyWallet(
         // Password
-        "",
+        password ? password : "",
```
Comment (TonyGiorgio) [inline src/logic/mutinyWalletSetup.ts:143]: `undefined`

**Comment by TonyGiorgio** [comment]
Existing password at the top should be removed if they're coming in from the "backup notification" flow. We know they're not going to have an existing password. 

Screenshot from 2023-07-10 14-19-13

**Comment by TonyGiorgio** [comment]
We'll probably have some users that forget their password. How should we handle that? Take them to the emergency page? 

Screenshot from 2023-07-10 14-22-35

**Comment by futurepaul** [comment]
> Existing password at the top should be removed if they're coming in from the "backup notification" flow. We know they're not going to have an existing password.

I agree this would be nice but requires a decent bit of logic to have a "referral page" so maybe just make an issue for now

**Comment by futurepaul** [comment]
Screenshot 2023-07-10 at 3 47 30 PM

**Review by TonyGiorgio — APPROVED** [review]

**Pull request merged**